packages/core/src/lib/translate/translate.ts

```diff
@@ -7,6 +7,8 @@ import vi from "../locale/vi.json";
 import hi from "../locale/hi.json";
 import ar from "../locale/ar.json";
 
+const devLanguage = "es"; // leave null or change into language code "en", "es"...
```

Comment:
Maybe switch wariable name to "allowOnlyLanguage"

Reply:
Sounds good!